clarify that Bazel team members should do their own PR routing (#179)
bazelbuild/bazel-website: maintainers-guide.md

## Lifecycle of a Pull Request

1. A user creates a pull request.
1. If you a member of a Bazel team and sending a PR against your own area,

>    you should pre-assigin your team label and finding the best reviewer.

Reviewer: ```suggestion
   you are responsible for assigning your team label and finding the best reviewer.
```

Author: done
